certchain/certmanager/ocsp_client.go

```diff
@@ -107,23 +107,23 @@ func (c OCSPClient) Fetch(chain *certchain.RawChain, now func() time.Time) (ocsp
 
 	raw, expiry, err := c.fetch(chain)
 	if err != nil {
-		err = fmt.Errorf("cannot fetch OCSP response: %v", err)
+		err = fmt.Errorf("cannot fetch OCSP response: %v, retrying in %v", err, retryWait)
```

Comment:
I also added the amount of time before next retry in the error, because it was useful to know how long the next retry would be.